draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1713,8 +1713,9 @@ data. The original JSON data is then used by the application. See
 * Do not disallow HMAC any longer.
 * Editorial changes aimed at improved clarity
 * Improve unlinkability considerations, mention that different KB keys must be used
+* Be more explicit that the VCDM and SD-JWT VC examples are only illustrative and do not define anything
 * Remove mention of unspecified key binding methods and the Enveloping SD-JWTs section
-
+* Update PID example
```

Comment:
I just happened to notice this was in the wrong draft revision when resolving conflicts merging in master

Reply:
Thanks, I seem to be bad at putting changelog entries in the right place.

Reply:
Maybe because I indented -08 differently than the others? It's my fault...